### PR TITLE
update test for enrichment service

### PR DESCRIPTION
**Pull request by wxli0**
relates to #481 point6
- update path
- Since object keys are unordered, use a helper to check for object equality

### CODE REVIEW

File: test/server/enrichment-map/enrichment/enrichment-test.js [hunk 1]
```diff
@@ -1,72 +1,90 @@
 const chai = require('chai');
 const expect = chai.expect;
-const {enrichment} = require('../../../server/enrichment-map/enrichment');
+const _ = require('lodash');
+const {enrichment} = require('../../../../src/server/enrichment-map/enrichment');
+
+
+// object keys are unordered
+const objectEquality = (obj1, obj2) => {
```
Comment (d2fong) [inline test/server/enrichment-map/enrichment/enrichment-test.js]: lodash should have some function that does this for you.

File: test/server/enrichment-map/enrichment/enrichment-test.js [hunk 2]
```diff
@@ -1,72 +1,90 @@
...
+  _.forEach(obj1, property => {
+    if (!obj2.has(property)) { return false; }
+    if (obj1.get(property) != obj2.get(property)) { return false; }
+  });
+  return true;
+};
+
 
 describe('test enrichment', function() {
+  this.timeout(500000);
```
Comment (d2fong) [inline test/server/enrichment-map/enrichment/enrichment-test.js]: why is this needed

Comment (wxli0) [reply test/server/enrichment-map/enrichment/enrichment-test.js]: Removing this line sometimes gives timeout error.

Comment (d2fong) [reply test/server/enrichment-map/enrichment/enrichment-test.js]: ok

File: test/server/enrichment-map/enrichment/enrichment-test.js (hunk 1 above)
Comment (d2fong) [reply test/server/enrichment-map/enrichment/enrichment-test.js]: probably see if something like this works:
https://lodash.com/docs/4.17.5#isEqual

Comment (wxli0) [reply test/server/enrichment-map/enrichment/enrichment-test.js]: value for property 't group' is different every time. Ignore the property when checking equality.
